src/command_modules/azure-cli-component/azure/cli/command_modules/component/commands.py

```diff
@@ -3,11 +3,13 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-#pylint: disable=line-too-long
+# python: disable=line-too-long
 
 from azure.cli.core.commands import cli_command
 
-cli_command(__name__, 'component list', 'azure.cli.command_modules.component.custom#list_components')
+cli_command(__name__, 'component list',
+            'azure.cli.command_modules.component.custom#list_components')
```

Comment:
This should be on a single line. Hence re-enabling the "line too long" for Pylint.